stable_baselines/common/base_class.py

```diff
@@ -153,6 +155,49 @@ def _setup_learn(self, seed):
         if seed is not None:
             set_global_seeds(seed)
 
+    @abstractmethod
+    def _get_parameter_list(self):
+        """
+        Get tensorflow Variables of model's parameters
+
+        This includes all variables necessary for continuing training (saving / loading).
+
+        :return: (list) List of tensorflow Variables
+        """
+        pass
+
+    def get_parameters(self):
+        """
+        Get current model parameters as dictionary of variable name -> ndarray.
+
+        :return: (dict) Dictionary of variable name -> ndarray of model's parameters.
```

Comment:
Why not an ordered dict?

Reply:
There is no reason to have it ordered apart from having consistency with the previous "lists in same order" method, which would then raise questions "Why this is OrderedDict?".

Reply:
Even though the ordered is preserved starting at python 3.6, I would find it cleaner to have an ordered dict.
That would also avoid potential bugs if people just want to recover numpy arrays using `param_dict.values()`.

And yes, in fact, `_get_parameter_list` is already kind of public through the keys of the parameters. So, I would put it public. Any thought on that @hill-a ?